frontend/src/views/CellGuide/components/CellGuideCard/components/MarkerGeneTables/index.tsx

```diff
@@ -504,37 +634,42 @@ const MarkerGeneTables = ({
     setPage(page);
   };
 
+  const selectedOptionOrgan = activeTable
+    ? selectedOrganFilterComputational
+    : selectedOrganFilterCanonical;
+  const handleChangeOrgan = activeTable
+    ? handleChangeOrganComputational
+    : handleChangeOrganCanonical;
+
+  const selectedOptionOrganism = selectedOrganismFilterComputational;
+  const handleChangeOrganism = handleChangeOrganismComputational;
```

Comment:
what's the reason behind this variable reassignment or can we just use `selectedOrganismFilterComputational` and `handleChangeOrganismComputational`?